modules/enterprise/gui/coregui/src/main/java/org/rhq/coregui/server/gwt/BundleGWTServiceImpl.java

```diff
@@ -71,7 +71,7 @@ public ResourceTypeBundleConfiguration getResourceTypeBundleConfiguration(int co
     public BundleVersion createOrStoreBundleVersionViaURL(String url, String username, String password)
         throws RuntimeException {
         try {
-            BundleVersion results = bundleManager.createBundleVersionViaURL(getSessionSubject(), url, username,
+            BundleVersion results = bundleManager.createOrStoreBundleVersionViaURL(getSessionSubject(), url, username,
```

Comment:
Yeah, sorry if I didn't explained well. I'm just pointing to other method, the one that is *supposed* to be called instead.

When I say *supposed*, is because:
 - The method match the method invocation, and
 - The method *bundleManager.createOrStoreBundleVersionViaURL* had traces of the *token* logic that is used on the 'Upload'.